src/openforms/variables/static_variables/static_variables.py

```diff
@@ -0,0 +1,16 @@
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from openforms.forms.constants import FormVariableDataTypes
+
+from ..base import BaseStaticVariable
+from ..registry import static_variables_register
+
+
+@static_variables_register("now")
+class Now(BaseStaticVariable):
+    name = _("Now")
+    data_type = FormVariableDataTypes.datetime
+
+    def get_initial_value(self, *args, **kwargs):
+        return timezone.now().isoformat()
```

Comment:
while implementing #1708, I noticed that I want access to the raw underlying datetime object and not the stringified version (so I can use the `date` and `time` template filters), so I'd drop the `.isoformat()`.

```suggestion
        return timezone.now()
```